internal/app/server/handler.go

```diff
@@ -1 +1,44 @@
 package server
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"parcel-service/internal/app/model"
+
+	"github.com/rs/zerolog/log"
+)
+
+func (s *server) createParcel(w http.ResponseWriter, r *http.Request) {
+	const (
+		CARRIER_FEE = 180.00
```

Comment:
this should be in service layer. later service will call pricing service (external service) to get those value